tests/configTests/extendEnv.test.js

```diff
@@ -0,0 +1,8 @@
+describe('Can access to env variables', () => {
+    test('access to .env.test', () => {
+        expect(process.env.NEXT_PUBLIC_GRAPHQL_URI).toBeDefined()
+    })
+    test('access to derived variables', () => {
+        expect(process.env.NEXT_PUBLIC_IS_USING_LOCAL_DATABSE).toBeDefined()
```

Comment:
can you also fix the typo to "DATABSE" (doing a research on all files because we probably reuse it)?